src/travel-details-screens/TripDetailsScreenComponent.tsx

```diff
@@ -221,11 +218,20 @@ function useGetTicketInfoFromTrip(tripPattern: TripPattern) {
   };
 }
 
-function waitTimeIsMoreThanAnHour(leg: Leg, nextLeg?: Leg) {
-  const waitTimeInSeconds = nextLeg
+function totalWaitTimeIsMoreThanAnHour(legs: Leg[]) {
+  return (
+    legs.reduce(
+      (sum, leg, currentIndex) =>
+        sum + getWaitTime(leg, legs[currentIndex + 1]),
```

Comment:
It is curious, this doesn't break the app because this is a clear index overflow here when the `currentIndex` is in the last element. But it was like that before.